source/sync/device-sync-deprecation.txt

```diff
@@ -7,11 +7,11 @@
 Device Sync Deprecation
 =======================
 
-.. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 2
-   :class: singlecol
+.. toctree::
+   :titlesonly:
+   :hidden:
+
+   Migrate to AWS AppSync </sync/migration/awsappsync>
 
 As of September 2024, Device Sync is deprecated for Atlas App Services. Device
 Sync will reach end-of-life and be removed on **September 30, 2025**. If you
```

Comment:
Can't suggest changes to the below line, but maybe update to specify `...you should migrate to one of the alternative solution providers listed below...`?